test/specs/calculator.add.multiply.spec.js

```diff
@@ -0,0 +1,158 @@
+        expect(calculator.add(-356, 125)).to.be.equal(-231);
+    });
+    it(`It should test the calculation with whole numbers`, () => {
+        expect(calculator.add(548, 0)).to.be.equal(548);
+    });
+    it(`It should test the calculation with fractions`, () => {
+        expect(calculator.add(3 / 8, 4 / 8)).to.be.equal(7 / 8);
+    });
+    it(`It should test the calculation with the rational numbers`, () => {
+        // eslint-disable-next-line max-len
+        expect(calculator.add(-245.969, 138.354)).to.be.equal(-107.61499999999998);
+    });
+});
+
+// eslint-disable-next-line max-len
+describe(`It should test Calculator class method 'add' negative scenarios`, () => {
+    let calculator;
+    beforeEach(() => {
+        calculator = new Calculator();
+    });
+    afterEach(() => {
+        calculator = null;
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'number' is null an error is thrown`, () => {
+        expect(() => calculator.add(null, 5, 6)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'increment' is not defined(null) an error is thrown`, () => {
+        expect(() => calculator.add(68, null)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'number' and 'increment' are not defined(null) an error is thrown`, () => {
+        expect(() => calculator.add(null, null)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'number' and 'increment' are not defined(values are empty) an error is thrown`, () => {
+        expect(() => calculator.add()()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if only 'number' is defined an error is thrown`, () => {
+        expect(() => calculator.add(5)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if only 'number' is defined(null) an error is thrown`, () => {
+        expect(() => calculator.add(null)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if only 'number' is defined(as undefined) an error is thrown`, () => {
+        expect(() => calculator.add(undefined)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'number' and 'increment' are not defined(undefined) an error is thrown`, () => {
+        expect(() => calculator.add(undefined, undefined)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'number' is not defined(undefined) an error is thrown`, () => {
+        expect(() => calculator.add(undefined, 5)()).to.throw();
+    });
+    // eslint-disable-next-line max-len
+    it(`It should test that if 'increment' is not defined(undefined) an error is thrown`, () => {
+        expect(() => calculator.add(68, undefined)()).to.throw();
```

Comment:
Please add checks to verify the Error message, not just the fact of throwing an Error.